context/app/static/js/components/entity-search/facets/FacetGroupAccordion/FacetGroupAccordion.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import { StyledExpandMoreIcon } from 'js/components/searchPage/filters/style';
+import { OuterAccordion, OuterAccordionSummary, OuterAccordionDetails } from './style';
+
+function FacetGroupAccordion({ label, isFirst, children }) {
+  return (
+    <OuterAccordion defaultExpanded={isFirst}>
```

Comment:
I'd suggest calling it `isExpanded`... Farther down, I wasn't sure why being first mattered. Sometimes, more abstraction is good, but since it's really just doing one thing, I think it'd be more clear to carry that through.